File: all-languages/flatiron/reverseString/reverseString.rb
def reverse_string1(str)
    newString = ""
    for i in (0..str.length-1).reverse_each
        newString += str[i]
    end
    newString
end
puts reverse_string1("paolo")


def reverse_string2(string)
    i = string.length - 1
    newString = ""
    while i >= 0
        newString += string[i]
        i -= 1
    end
    newString
end
puts reverse_string2("paolo")

File: various/flatiron/reverseString/reverseString.py

def reverse_string1(string) :
    i = len(string) - 1
    new_string = ""
    while i >= 0 :
        new_string += string[i]
        i -= 1
    return new_string
print(reverse_string1("paolo"))

def reverse_string2(string):
    newString = ""
    for i in range(len(string) -1, -1, -1):
        newString += string[i]
    return newString
print(reverse_string2("paolo"))